tests/test_app.py

```diff
@@ -6,10 +6,13 @@
     generate_test_authorization,
     generate_static_public_address,
 )
+import settings
 
 
 class TestApi(TestCase):
     REMOTE_ADDR = '8.8.8.8'
+    settings.NODE_WHITELIST_ENABLED = False
```

Comment:
It should be easy to test that decorator actually works - you can pass `remote_addr` param to `self._post` to test requests with different ips.

Reply:
tests now are there